sbi/neural_nets/mnle.py

```diff
@@ -309,7 +309,7 @@ def log_prob(self, x: Tensor, context: Tensor) -> Tensor:
             # Transform to log-space if needed.
             torch.log(cont_x) if self.log_transform_x else cont_x,
             # Pass parameters and discrete x as context.
-            context=torch.cat((context, disc_x), dim=1),
+            condition=torch.cat((context, disc_x), dim=1),
```

Comment:
should not this still be `context` because MNLE was not updated to use `DensityEstimator` yet?